doc/semicons.xml

```diff
@@ -299,3 +299,174 @@ true]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="StrongSemilatticeOfSemigroups">
+  <ManSection>
+    <Func Name="StrongSemilatticeOfSemigroups" Arg="D, L, H"/>
```

Comment:
This is an operation, not a global function, so it should be labelled/documented as such:
```suggestion
    <Oper Name="StrongSemilatticeOfSemigroups" Arg="D, L, H"/>
```
The analogous thing applies for many of your other bits of documentation.